src/localization/localization.ts

```diff
@@ -49,13 +48,33 @@ const localization = {
             propStorybookStoriesInfo: "Names of stories defined under `.stories.js` of a component."
         },
         config: {
+            custom: {
+                setConfig: "Set as Zeplin configuration file",
+                noFileSelected: "No file selected.",
+                alreadySet: "Selected file is already a configuration file.",
+                fileNotSaved: "Selected file is not saved, please save it before continuing.",
+                fileNotValid: "Selected file is not valid as a Zeplin configuration, please review it before continuing.",
+                set: "File set as Zeplin configuration. The root is still the same. Would you like to select a custom root?",
+                defaultConfigRootCannotBeChanged: "You cannot change default Zeplin configuration file's root.",
+                nonConfigSelected: "Selected file is not a Zeplin configuration.",
+                currentRoot: (path: string) => `Zeplin root: ${path}`,
+                setRoot: "Set root for Zeplin configuration file",
+                defaultConfigCannotBeUnset: "You cannot unset a default Zeplin configuration file.",
+                rootSet: "Folder set as root for Zeplin configuration file",
+                unset: "File unset as Zeplin configuration."
+            },
+            update: {
+                rootSet: "Folder set as root for Zeplin configuration file",
+                moved: "Zeplin configuration file moved.",
+                removed: "Zeplin configuration file removed."
```

Comment:
I'm not sure if there's a reason why some of these have a period at the end and some don't, but just wanted to give you a heads up!

Reply:
that's because some are picker titles and some other error/info messages, but apparently some of them are wrong, thanks for noticing.